package/raphael.js

```diff
@@ -672,27 +672,27 @@ module.exports = function (key) {
 /* 17 */
 /***/ (function(module, exports) {
 
-var g;
-
-// This works in non-strict mode
-g = (function() {
-	return this;
-})();
-
-try {
-	// This works if eval is allowed (see CSP)
-	g = g || Function("return this")() || (1,eval)("this");
-} catch(e) {
-	// This works if the window reference is available
-	if(typeof window === "object")
-		g = window;
-}
-
-// g can still be undefined, but nothing to do about it...
-// We return undefined, instead of nothing here, so it's
-// easier to handle this case. if(!global) { ...}
-
-module.exports = g;
+var g;
+
+// This works in non-strict mode
+g = (function() {
+	return this;
+})();
+
+try {
+	// This works if eval is allowed (see CSP)
+	g = g || Function("return this")() || (1,eval)("this");
+} catch(e) {
+	// This works if the window reference is available
+	if(typeof window === "object")
+		g = window;
+}
+
+// g can still be undefined, but nothing to do about it...
+// We return undefined, instead of nothing here, so it's
+// easier to handle this case. if(!global) { ...}
+
+module.exports = g;
```

Comment:
@KaushalKumarChaudhary What has been changed in this file? Why is this part of your commit?

Reply:
touch-action css style had been modified here. If this file is auto generated then it can be reverted. Please confirm.

Reply:
Am not clear. If I compare side by side I do not see any visible code change in the above snippet !!

Reply:
yes, there is no change in above snippet. Not sure why git picked it, but this will not create any issue as the code is same.

Reply:
Then this is junk. You do not commit files where there is no change. This is probably due to change in line-ending character. As developer you commit only the files required, to keep the commit clean and history proper. Check for your line ending setting in your IDE.

Reply:
This file is raphael.js and it has changes as mentioned above for touch-action css styles and hence committed.